using-mobile-extensions/mobile-core/rules-engine/rules-engine-details.md

```diff
@@ -373,25 +375,20 @@ If the conditions pass, the in-app message is displayed \(first consequence\), a
 }
 ```
 
-## Rules URLs
+## Rules URL
 
-Since the Rules Engine can support multiple end-point URLs from where to fetch rules, the URLs are prioritized based on the following scheme:
+The Rules Engine requires an endpoint URL to be configured in the configuration, which specifies the location from where rules should be fetched:
 
 ```text
 {
-   "rules.url" : [
-        "https://url/endpoint1",
-        "https://url/endpoint2",
-        ..
-    ]
+   "rules.url" : "https://assets.adobedtm.com/example.zip"
 }
 ```
 
-The Adobe Experience Platform SDKs process the URLs based on the order in which they are specified. Both endpoints are URLs that point to a zipped rules collection that contains a `rules.json` file, an asset folder that contains images, and HTMLs that are used by the rules. After the rules are downloaded and extracted, the contents of this compressed file are stored in the cache.
+The Adobe Experience Platform SDKs process the URL that points to a zipped rules collection, which contains a `rules.json` file. After the rules are downloaded and extracted, the contents of this compressed file are stored in the cache.
```

Comment:
Please reword the 1st sentence as follows:
"The Adobe Experience Platform SDKs process the URL that points to a zipped rules collection, and this collection contains a `rules.json` file."